src/test/java/com/scottlogic/filters/KeywordContentPostFilterTest.java

```diff
@@ -43,42 +47,70 @@ void filter_EmptyList_EmptyList() {
 
     @Test
     void filter_1ItemList_1ItemList() {
-        List<UserPost> inputList = List.of(userPost1);
-        List<UserPost> expected = List.of(userPost1);
+        List<UserPost> inputList = List.of(userPost_Hello);
+        List<UserPost> expected = List.of(userPost_Hello);
 
         List<UserPost> actual = new KeywordContentPostFilter("Hello").filter(inputList);
 
         assertEquals(expected, actual);
     }
 
+    @Test
+    void filter_1Item0Match_EmptyList() {
+        List<UserPost> inputList = List.of(userPost_Hello);
+        List<UserPost> expected = List.of();
+
+        List<UserPost> actual = new KeywordContentPostFilter("Test").filter(inputList);
```

Comment:
A small thing, tests are all about readability and how quickly you can see why the error occurred this means sometimes factoring things to be across multiple lines can help to make the test clearer. For example instead of this line you could split the filter into a separate line like 
```KeywordContentPostFilter filter_Test = new KeywordContentPostFilter("Test")``` 
and then use the `filter_Test` on this line can make it more obvious what the filter being used is as it is closer to the content of the list and thus easier to cross reference.

Reply:
Yeah that makes sense